Wordle.Api/Wordle.Api/Controllers/WordController.cs

```diff
@@ -6,22 +6,21 @@ namespace Wordle.Api.Controllers;
 [Route("[controller]")]
 public class WordController(WordOfTheDayService wordOfTheDayService) : ControllerBase
 {
-    [HttpGet("RandomWord")]
-    public async Task<string> GetRandomWord()
-    {
-        var randomWord = await wordOfTheDayService.GetRandomWord();
-        return randomWord.Text;
-    }
+	[HttpGet("RandomWord")]
+	public string GetRandomWord()
+	{
+		return wordOfTheDayService.GetRandomWord();
+	}
 
-    /// <summary>
-    /// Get the word of the day.
-    /// </summary>
-    /// <param name="offsetInHours">Timezone offset in hours. Default to PST</param>
-    /// <returns></returns>
-    [HttpGet("WordOfTheDay")]
-    public async Task<string> GetWordOfDay(double offsetInHours = -7.0)
-    {
-        DateOnly today = DateOnly.FromDateTime(DateTime.UtcNow.AddHours(offsetInHours));
-        return await wordOfTheDayService.GetWordOfTheDay(today);
-    }
-}
+	/// <summary>
+	/// Get the word of the day.
+	/// </summary>
+	/// <param name="offsetInHours">Timezone offset in hours. Default to PST</param>
+	/// <returns></returns>
+	[HttpGet("WordOfTheDay")]
+	public async Task<string> GetWordOfDay(double offsetInHours = -7.0)
+	{
+		DateOnly today = DateOnly.FromDateTime(DateTime.UtcNow.AddHours(offsetInHours));
+		return await wordOfTheDayService.GetWordOfTheDayAsync(today);
+	}
+}
```

Comment:
hint for word by date api call `[HttpGet("WordByDate/{month}/{day}/{year}")]
    public async Task<string?> GetWordByDate(int month, int day, int year)`

Reply:
Thanks Spencer! Will be implementing it very soon